packages/mobile/App/ui/navigation/screens/home/PatientDetails/GeneralInfo.tsx

```
@@ -29,25 +34,24 @@ export const GeneralInfo = ({ onEdit, patient }: GeneralInfoProps): ReactElement
     [PATIENT_DATA_FIELDS.EMAIL, patient.email],
-    [
+  ];
+
+  if (!isUsingHierarchyLogic) {
+    fields.push([
       PATIENT_DATA_FIELDS.VILLAGE_ID,
       <TranslatedReferenceData
         fallback={patient.village?.name ?? ''}
         value={patient.village?.id}
         category="village"
       />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)